test/e2e-tests.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
If this gets more complex, we could also run this in go and then use exec.Command. https://github.com/grafeas/kritis/blob/master/integration/run_test.go#L146

Sorry, shd have mentioned this before.

Reply:
hmm after looking at the comments, looks like this script is prow specific and needs to be there